interpreter/text/arrange.ml

```diff
@@ -648,30 +634,19 @@ let module_with_var_opt x_opt (m, cs) =
   let mx = ref 0 in
   let gx = ref 0 in
   let imports = list (import fx tx mx gx) m.it.imports in
-  let open Custom in
-  Node ("module" ^ var_opt x_opt,
-    let secs, cs = iterate (custom_section m (Before Type)) cs in secs @
```

Comment:
True, now the custom annotations are all at the end.
But it could be improved by some more complex logic in the "arrange" function of the handler, if we really want.